src/data_structures.rs

```diff
@@ -65,7 +66,7 @@ pub trait PCRandomness: Clone {
 
 /// Defines the minimal interface of evaluation proofs for any polynomial
 /// commitment scheme.
-pub trait PCProof: Clone + algebra_core::ToBytes {
+pub trait PCProof: Clone + ToBytes {
     /// Size in bytes
     fn size_in_bytes(&self) -> usize;
```

Comment:
Since we got `CanonicalSerialize` now, this method could be replaced with the `serialized_size` method's implementation. Same comment applies to `size_in_bytes` above in `PcCommitment`